libs/blocks/global-navigation/global-navigation.js

```diff
@@ -163,14 +207,23 @@ const closeOnClickOutside = (e) => {
 class Gnav {
   constructor(body, el) {
     this.blocks = {
-      profile: {
-        rawElem: body.querySelector('.profile'),
-        decoratedElem: toFragment`<div class="feds-profile"></div>`,
-      },
       search: { config: { icon: CONFIG.icons.search } },
       breadcrumbs: { wrapper: '' },
     };
 
+    // TODO: metadata + project config
+    this.universalNavComponents = getMetadata('universal-nav')?.toLowerCase()?.split(',').map((option) => option.trim())
```

Comment:
Do we need anything else in terms of the authoring for this? Or is a list of the components in the metadata (+ the on toggle) enough?

Reply:
the list of components should be good enough, but I'm thinking about enhancing the components, e.g. providing custom fields in Profile. If this does not turn out to be a requirement I will remove the todo.